api/appStore/InstalledAppRestHandler.go

```diff
@@ -411,31 +411,26 @@ func (handler *InstalledAppRestHandlerImpl) FetchNotesForArgoInstalledApp(w http
 		return
 	}
 	handler.Logger.Infow("request payload, FetchNotesForArgoInstalledApp, app store", "installedAppId", installedAppId, "envId", envId)
-
-	notes, appName, err := handler.installedAppService.FindNotesForArgoApplication(installedAppId, envId)
+	notes, err := handler.installedAppService.FetchNotesFromdb(installedAppId, envId, token, handler.checkNotesAuth)
```

Comment:
change the function name, fetchChartNotes

Reply:
done